app/app.js

```diff
@@ -623,9 +630,11 @@ app.post('/snap', [
     const duration = ((Date.now() - startTime) / 1000);
 
     if (err) {
-      lgParams.duration = duration
-      log.warn(lgParams, `Snap FAILED in ${duration} seconds. ${err}`);
-      res.status(500).send('' + err);
+      lgParams.fail = true;
```

Comment:
Production is already failing, are you sure you need to explicitly set `fail: true` as well? ;-)

Reply:
Just trying to be extra sure that it fails next time!